device-types/Cisco/ISR-1127-8PLTEP.yaml

```diff
@@ -0,0 +1,43 @@
+---
+manufacturer: Cisco
+model: ISR 1127-8PLTEP
+part_number: C1127-8PLTEP
+slug: cisco-isr-1127-8pltep
+weight: 4.53
+weight_unit: lb
+u_height: 1
+is_full_depth: false
+interfaces:
+  - name: GigabitEthernet0/0/0
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/1
+    type: 1000base-t
+  - name: GigabitEthernet0/1/0
+    type: 1000base-t
+  - name: GigabitEthernet0/1/1
+    type: 1000base-t
+  - name: GigabitEthernet0/1/2
+    type: 1000base-t
+  - name: GigabitEthernet0/1/3
+    type: 1000base-t
+  - name: GigabitEthernet0/1/4
+    type: 1000base-t
+  - name: GigabitEthernet0/1/5
+    type: 1000base-t
+  - name: GigabitEthernet0/1/6
+    type: 1000base-t
+  - name: GigabitEthernet0/1/7
+    type: 1000base-t
+module-bays:
+  - name: Cellular
+    position: '1'
```

Comment:
Is this "1" really useful here ? There is only one slot. Is there another ISR model with multiple slots ? 
Usually this identifier is used in module-types as part of their name or components using templated strings like such: ge-0/{module}/0